src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -192,6 +192,11 @@ protected void handlePut(final RestChannel channel, final RestRequest request, f
 
 		final SecurityDynamicConfiguration<?> existingConfiguration = load(getConfigName(), false);
 
+		if (existingConfiguration.getSeqNo() < 0) {
+		    forbidden(channel, "Security index need to be updated to support '" + getConfigName().toLCString() + "'. Use OpenDistroSecurityAdmin to populate.");
```

Comment:
> couldn't understand this case.
> When this (existingConfiguration.getSeqNo() < 0) can happen?
> 
> And are you suggesting Security Admin tool to initialize the index?

Correct. The need comes from nodesdn usecase. This is being introduced in older versions and hence it should be backward compatible, which means in the cases where the index is already created, the APIs will not work out of the box and will need security admin to create the document in index.

Since the need is there, I added this generic check to provide better error message rather than ending with 5xx.

Reply:
> When this (existingConfiguration.getSeqNo() < 0) can happen?

This will happen when the index doesn't exist. ConfigurationLoader handles empty scenario and returns an object with negative seq number.